### ISSUE TITLE
Request for maintainer(s)

### ISSUE THREAD

**Issue by darvid** [darvid/python-hyperscan#44]
### Discussed in https://github.com/darvid/python-hyperscan/discussions/43

<div type='discussions-op-text'>

<sup>Originally posted by **darvid** July 24, 2022</sup>
I'm struggling with some personal issues right now and probably won't have the time to maintain this package in the near future. If someone is willing to become a maintainer please let me know. Thanks.</div>

**Comment by deronnax** [comment]
seems like no one stepped in, but on the other hand, the project seems to be pretty popular, so if you enable Github Sponsoring on this project, you might get a very surprizing positive outcome (💵 💵 💵 ).

**Comment by darvid** [comment]
@markos yes, thank you for the interest and definitely sounds like you'd be the most equipped to support development of this project. I have slightly more bandwidth now compared to when I made the initial request, so I can probably continue to help in an approval/release/reviews/etc. capacity if it helps reduce the overhead of a complete ownership transfer. any help with features/bugfixes would be appreciated, and I'm still personally interested and motivated in continuing to develop/support the package (, I just have had a lot going on in life recently.

but open to anything, also happy to chat. definitely think it makes sense to add you/the vectorscan maintainers as maintainer of this package, though. and yes given the license change I think it's time to move to vectorscan

**Comment by ozturkberkay** [comment]
Hey @darvid! We couldn't access the repo recently, but we are glad it's back online now. I'm curious what are your plans for the project? Should we expect a migration to Vectorscan?

**Comment by darvid** [comment]
@ozturkberkay thanks and apologies for the downtime, my account was temporarily suspended for a security alert, but Github staff were able to restore it after the new year holiday thankfully.  😅

and yes the plan is to migrate to vectorscan, I will hopefully have time in the next couple of weeks for a first pass at updating manylinux-hyperscan (the underlying build environment container) and then the build scripts for the extension.
